tensorflow/tflite-micro
# Update CI Dockerfile

BUG=1891

Update Dockerfile.micro to install QEMU.  This allows for CI to run QEMU based tests.

Verification:
1) Build container `docker build -f ci/Dockerfile.micro -t tflm-ci . `
2) Verify `qemu-arm` exists as  `/usr/local/qemu/qemu-arm`
3) Run the container with the tflm directory mounted as a volume
  `docker run -v $(pwd):/tmp/tflm -it tflm-ci bash`
 4. Inside the tflm root in the container, run the qemu ci script.
   `tensorflow/lite/micro/tools/ci_build/test_cortex_m_qemu.sh`
